sdcm/cluster.py

```diff
@@ -3652,7 +3688,8 @@ def restart_scylla_monitoring(self, sct_metrics=False):
               message="Waiting for reconfiguring scylla monitoring")
     def reconfigure_scylla_monitoring(self):
         for node in self.nodes:  # pylint: disable=no-member
-            db_targets_list = [n.ip_address for n in self.targets["db_cluster"].nodes]  # node exporter + scylladb
+            # node exporter + scylladb
```

Comment:
this comment isn't correct, can be removed

Reply:
removed